trace-token/src/main/java/io/airlift/tracetoken/TraceTokenManager.java

```diff
@@ -15,10 +15,16 @@
  */
 package io.airlift.tracetoken;
 
-import java.util.UUID;
+import java.util.concurrent.atomic.AtomicLong;
+
+import static java.util.Locale.ENGLISH;
+import static java.util.UUID.randomUUID;
 
 public class TraceTokenManager
 {
+    private final String prefix = randomUUID().toString().toLowerCase(ENGLISH).replace("-", "") + ":";
```

Comment:
We don't need the `:` and it means the value can be used in any place that can tolerate hex characters